.github/dependabot.yml

```diff
@@ -5,3 +5,15 @@ updates:
     directory: "/"
     schedule:
       interval: "weekly"
+
+  # Maintain dependencies for GitHub Actions
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+
+  # Maintain dependencies for Docker images
+  - package-ecosystem: "docker"
```

Comment:
I don't think this is a good idea, this might break our addons or older kuberentes versions